cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -109,6 +113,8 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 			// End of Github Actions foldable section annotation.
 			fmt.Println("::endgroup::")
 		}
+		// Call e2eCleanUp to close the SpanReader and SpanWriter gRPC connection.
```

Comment:
The way it's currently written it has a bunch of requires which could panic and prevent the other cleanups. I suggest replacing requires with error checks and logging.

Call to e2e cleanup is better done before printing logs.

Reply:
I've replaced the require statements with error checks and logs which i think might prevent the other cleanups when panicked. Let me know if this looks fine.